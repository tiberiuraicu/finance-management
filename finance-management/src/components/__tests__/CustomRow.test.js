import CustomRow from "../CustomRow";
import { shallow } from 'enzyme';

describe('CustomRow component tests', () => {
    it('renders the component', () => {
      const wrapper = shallow(<CustomRow />);
      expect(wrapper).toMatchSnapshot();
    });
  });