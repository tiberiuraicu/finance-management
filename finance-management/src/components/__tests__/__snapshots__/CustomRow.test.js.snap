// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomRow component tests renders the component 1`] = `ShallowWrapper {}`;
